docker-compose.bridge.yml

```diff
@@ -0,0 +1,199 @@
+version: "3.7"
```

Comment:
This file is same as docker-compose.yml. Does is make sense to keep both of them around?